### PR TITLE
validation for atlease one problem or mcq on create/edit drive …

### PR DESCRIPTION
**Pull request by somnath77**
…form

### CODE REVIEW

File: src/modules/admin/createDrive/CreateDriveCotainer/index.jsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ const CreateDriveContainer = () => {
   const schema = yup.object().shape({
     name: yup.string().required(),
     description: yup.string().required(),
-    // drives_problems_attributes: yup.array().min(1, 'Please select problem(s)'),
+    problems_count: yup.number().min(1, 'Please select at least on of problem(s) or MCQ(s)'),
```
Comment (onkar-josh) [inline src/modules/admin/createDrive/CreateDriveCotainer/index.jsx:85]: should be Please select at least one of problem(s) or MCQ(s)

File: src/modules/admin/editDrive/EditDriveCotainer/index.jsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ const EditDriveContainer = () => {
   const schema = yup.object().shape({
     name: yup.string().required(),
     description: yup.string().required(),
-    currentProblems: yup.array().min(1, 'Please select problem(s)'),
+    problems_count: yup.number().min(1, 'Please select at least on of problem(s) or MCQ(s)'),
```
Comment (onkar-josh) [inline src/modules/admin/editDrive/EditDriveCotainer/index.jsx:90]: same as above

File: src/modules/admin/createDrive/CreateDriveCotainer/index.jsx (hunk 1 above)
Comment (somnath77) [reply src/modules/admin/createDrive/CreateDriveCotainer/index.jsx]: done

File: src/modules/admin/editDrive/EditDriveCotainer/index.jsx (hunk 2 above)
Comment (somnath77) [reply src/modules/admin/editDrive/EditDriveCotainer/index.jsx]: done

**Review by onkar-josh — APPROVED** [review]

**Pull request merged**